heartbeat/docs/reference/configuration/heartbeat-options.asciidoc

```diff
@@ -0,0 +1,448 @@
+[[configuration-heartbeat-options]]
+=== Heartbeat Configuration
+
+The `heartbeat` section of the +heartbeat.yml+ config file specifies the
+list of `monitors` that Heartbeat uses to ping your remote hosts to
+determine if they are available. Each `monitor` item begins with a dash (-) and
+specifies the type of monitor to use, the hosts to ping, and other settings
+that control ping behavior. 
+
+The following example configures two monitors, an `icmp` monitor and a `tcp`
+monitor:
+
+//I use the same example here that I use in the config section, but I really think it should probably be different. Suggestions? I do think that we should show more than one monitor in the example, even though it makes the example a bit longer, to show that you can specify more than one monitor.
+
+[source,yaml]
+----------------------------------------------------------------------
+heartbeat.monitors:
+- type: icmp
+  name: icmp
+  enabled: true
+  schedule: '*/5 * * * * * *'
+  hosts: ["myhost"]
+  ipv4: true
+  ipv6: true
+  mode: any
+  timeout: 16s
+  wait: 1s
+- type: tcp 
+  name: tcp
+  enabled: true
+  schedule: '@every 5s'
+  hosts: ["myhost:12345"]
+  ipv4: true
+  ipv6: true
+  mode: any
+  timeout: 16s
+heartbeat.scheduler:
+  location: 'UTC-08:00'
+output.elasticsearch:
+  hosts: ["myhost:9200"]
+----------------------------------------------------------------------
```

Comment:
Yeah, feel free to include

```
heartbeat.scheduler:
  limit: 10
```

setting limit to 10 guarantees only 10 concurrent IO-tasks being active. An IO-task can be the actual ping or resolving an address via DNS.